src/utils/py/daos_api.py

```diff
@@ -1894,7 +1894,7 @@ class DaosContext(object):
     def __init__(self, path):
         """ setup the DAOS API and MPI """
 
-        self.libdaos = ctypes.CDLL(path+"libdaos.so.0.0.2",
+        self.libdaos = ctypes.CDLL(path+"libdaos.so.0.4.0",
```

Comment:
Is there some way we can automatically pick the the version change, instead of keep changing this every time.